### PR TITLE
[Verification] Adds event ids to InvalidEventsCollection error

### PR DESCRIPTION
**Pull request by yhassanzadeh13**
Adds event ids to `InvalidEventsCollection` error.

### CODE REVIEW

File: model/chunks/chunkFaults.go [hunk 1]
```diff
@@ -107,7 +109,8 @@ func (c *CFInvalidEventsCollection) ExecutionResultID() flow.Identifier {
 }
 
 func (c *CFInvalidEventsCollection) String() string {
-	return fmt.Sprintf("events collection hash differs, got %x expected %x for chunk %d with result ID %s", c.computed, c.expected, c.chunkIndex, c.resultID)
+	return fmt.Sprintf("events collection hash differs, got %x expected %x for chunk %d with result ID %s, events IDs: %v", c.computed, c.expected,
```
Comment (zhangchiqing) [inline model/chunks/chunkFaults.go:112]: does `%v` print each event ID?

Comment (yhassanzadeh13) [reply model/chunks/chunkFaults.go:112]: yes, example error would be like this: 
```
events collection hash differs, got a8f693a7c559d2d0b35db9cceb87e598ae9d6aa31b1a701b9e4ba8f1c5df3c88 expected 33d76ebd6c705749658a7f14468974d4d75179ed61ce34ab30f16bc083ee5d0e for chunk 0 with result ID 78c34808327e356f1c3fc7b0411d804cbcfd9b62393b9c78538d54c686b53ff6, events IDs: [ccf116994d6a6997aa21e9a009e11aa5ffb1e0a92b727bbe8a42b09d630789f5 276088fd7b1177e3cd820179490262a71fcf2ea08f11c92a71c78727152885dc ccf116994d6a6997aa21e9a009e11aa5ffb1e0a92b727bbe8a42b09d630789f5 276088fd7b1177e3cd820179490262a71fcf2ea08f11c92a71c78727152885dc ccf116994d6a6997aa21e9a009e11aa5ffb1e0a92b727bbe8a42b09d630789f5 276088fd7b1177e3cd820179490262a71fcf2ea08f11c92a71c78727152885dc ccf116994d6a6997aa21e9a009e11aa5ffb1e0a92b727bbe8a42b09d630789f5 276088fd7b1177e3cd820179490262a71fcf2ea08f11c92a71c78727152885dc 5f057d6b1873204862a621f00a5d49e016dc40cfb98c1a699ff1b08db06406ae ccf116994d6a6997aa21e9a009e11aa5ffb1e0a92b727bbe8a42b09d630789f5 276088fd7b1177e3cd820179490262a71fcf2ea08f11c92a71c78727152885dc]

```

Comment (m4ksio) [reply model/chunks/chunkFaults.go:112]: I don't think event IDs tells us enough, we ideally want to dump entire events list, with all the fields

Comment (zhangchiqing) [reply model/chunks/chunkFaults.go:112]: yeah, we could add both.

If events are all the same, but event ID is different, then the problem is in the calculation of the event hash.
If events are different, then the problem is in the either execution or somewhere else.

**Review by zhangchiqing — APPROVED** [review]

File: model/chunks/chunkFaults.go (hunk 1 above)
Comment (yhassanzadeh13) [reply model/chunks/chunkFaults.go]: Added in https://github.com/onflow/flow-go/pull/1042/commits/d1a617c3c21d85d7ae9d91b0af5d4c7e06015017 now, error will look like:
```
events collection hash differs, got a8f693a7c559d2d0b35db9cceb87e598ae9d6aa31b1a701b9e4ba8f1c5df3c88 expected 33d76ebd6c705749658a7f14468974d4d75179ed61ce34ab30f16bc083ee5d0e for chunk 0 with result ID 414876ad8655805ade2dd5ec66ed7f0141971ccac18e1952c5f1f3c099967ea5, events IDs: [ccf116994d6a6997aa21e9a009e11aa5ffb1e0a92b727bbe8a42b09d630789f5 276088fd7b1177e3cd820179490262a71fcf2ea08f11c92a71c78727152885dc ccf116994d6a6997aa21e9a009e11aa5ffb1e0a92b727bbe8a42b09d630789f5 276088fd7b1177e3cd820179490262a71fcf2ea08f11c92a71c78727152885dc ccf116994d6a6997aa21e9a009e11aa5ffb1e0a92b727bbe8a42b09d630789f5 276088fd7b1177e3cd820179490262a71fcf2ea08f11c92a71c78727152885dc ccf116994d6a6997aa21e9a009e11aa5ffb1e0a92b727bbe8a42b09d630789f5 276088fd7b1177e3cd820179490262a71fcf2ea08f11c92a71c78727152885dc 5f057d6b1873204862a621f00a5d49e016dc40cfb98c1a699ff1b08db06406ae ccf116994d6a6997aa21e9a009e11aa5ffb1e0a92b727bbe8a42b09d630789f5 276088fd7b1177e3cd820179490262a71fcf2ea08f11c92a71c78727152885dc] events: [event.someType: ccf116994d6a6997aa21e9a009e11aa5ffb1e0a92b727bbe8a42b09d630789f5 event.otherType: 276088fd7b1177e3cd820179490262a71fcf2ea08f11c92a71c78727152885dc event.someType: ccf116994d6a6997aa21e9a009e11aa5ffb1e0a92b727bbe8a42b09d630789f5 event.otherType: 276088fd7b1177e3cd820179490262a71fcf2ea08f11c92a71c78727152885dc event.someType: ccf116994d6a6997aa21e9a009e11aa5ffb1e0a92b727bbe8a42b09d630789f5 event.otherType: 276088fd7b1177e3cd820179490262a71fcf2ea08f11c92a71c78727152885dc event.someType: ccf116994d6a6997aa21e9a009e11aa5ffb1e0a92b727bbe8a42b09d630789f5 event.otherType: 276088fd7b1177e3cd820179490262a71fcf2ea08f11c92a71c78727152885dc event.Extra: 5f057d6b1873204862a621f00a5d49e016dc40cfb98c1a699ff1b08db06406ae event.someType: ccf116994d6a6997aa21e9a009e11aa5ffb1e0a92b727bbe8a42b09d630789f5 event.otherType: 276088fd7b1177e3cd820179490262a71fcf2ea08f11c92a71c78727152885dc]
=== RUN   TestChunkVerifier/TestFailedTx
```

**Comment by yhassanzadeh13** [comment]
This PR already addressed in https://github.com/onflow/flow-go/pull/1048/files

**Pull request closed without merging**